aztk/spark/helpers/create_cluster.py

```diff
@@ -211,7 +211,7 @@ def generate_cluster_start_task(
     ] + __get_docker_credentials(spark_client)
 
     # start task command
-    command = __cluster_install_cmd(zip_resource_file, gpu_enabled, docker_repo, worker_on_master, file_shares, mixed_mode)
+    command = __cluster_install_cmd(zip_resource_file, gpu_enabled, docker_repo, plugins, worker_on_master, file_shares, mixed_mode)
```

Comment:
hhm weird, not sure how I managed to create a cluster with that problem

Reply:
this was almost certainly a merge error that slipped in, should have been caught in review.